internal/usage/usage.go

```diff
@@ -138,7 +138,10 @@ dbName and collection are required only for built-in roles.`
 	NotificationsChannelName                  = "Slack channel name. Required for the SLACK notifications type."
 	AlertConfigAPIKey                         = "Datadog API Key, Opsgenie API Key, VictorOps API key." //nolint:gosec // This is just a message not a credential
 	APIKey                                    = "API Key."
-	RoutingKey                                = "An optional field for your Routing Key."
+	DatadogAPIKey                             = "Datadog API key that allows Atlas to access your Datadog account."
```

Comment:
add 
`//nolint:gosec // This is just a message not a credential` at the end of the line or remove, depending on the complain from the linter

Reply:
@gssbzn Fixed it and the linter no longer shows errors, thank you!